cfme/common/provider_views.py

```diff
@@ -94,16 +95,29 @@ def __getattribute__(self, item):
 
     @property
     def is_displayed(self):
-        if self.context['object'].appliance.version >= '5.7':
-            subtitle = 'Summary' if self.toolbar.view_selector.selected == 'Summary View' \
-                else 'Dashboard'
+        if (not self.toolbar.view_selector.is_displayed or
+                self.toolbar.view_selector.selected == 'Summary View'):
+            subtitle = 'Summary'
         else:
-            subtitle = 'Summary'  # 5.6 has only only Summary view
-        title = '{name} ({subtitle})'.format(name=self.context['object'].name, subtitle=subtitle)
+            subtitle = 'Dashboard'
 
-        return self.logged_in_as_current_user and \
-            self.navigation.currently_selected == ['Compute', 'Infrastructure', 'Providers'] and \
-            self.breadcrumb.active_location == title
+        title = '{name} ({subtitle})'.format(name=self.context['object'].name,
+                                             subtitle=subtitle)
+        return self.logged_in_as_current_user and self.breadcrumb.active_location == title
+
+
+class InfraProviderDetailsView(ProviderDetailsView):
```

Comment:
I tried to move these classes to respective modules but I ran into issue with cyclical imports. 
after discussing this with Pete, we decided to leave them as is and move everything to common.ui when all providers are completely converted.